proto/ibc/applications/transfer/v1/tx.proto

```diff
@@ -52,7 +52,7 @@ message MsgTransfer {
   // tokens to be transferred
   repeated cosmos.base.v1beta1.Coin tokens = 9 [(gogoproto.nullable) = false];
   // optional forwarding information
-  ForwardingInfo forwarding_path = 10;
+  Forwarding forwarding_path = 10;
```

Comment:
I'd suggest also renaming `forwarding_path` to `forwarding`. I think it reads more naturally with the field name also. E.g. `forwarding.Hops` etc 

Curious to hear what others think?